rhea/build/boards/altera/_de1_SoC.py

```diff
@@ -0,0 +1,229 @@
+                                'AG25', # GPIO_1[9]
+                                'AG26',  # GPIO_1[10]
+                                'AH24', # GPIO_1[11]
+                                'AH27', # GPIO_1[12]
+                                'AJ27', # GPIO_1[13]
+                                'AK29', # GPIO_1[14]
+                                'AK28', # GPIO_1[15]
+                                'AK27', # GPIO_1[16]
+                                'AJ26', # GPIO_1[17]
+                                'AK26', # GPIO_1[18]
+                                'AH25', # GPIO_1[19]
+                                'AJ25', # GPIO_1[20]
+                                'AJ24', # GPIO_1[21]
+                                'AK24', # GPIO_1[22]
+                                'AG23', # GPIO_1[23]
+                                'AK23', # GPIO_1[24]
+                                'AH23', # GPIO_1[25]
+                                'AK22', # GPIO_1[26]
+                                'AJ22', # GPIO_1[27]
+                                'AH22', # GPIO_1[28]
+                                'AG22', # GPIO_1[29]
+                                'AF24', # GPIO_1[30]
+                                'AF23', # GPIO_1[31]
+                                'AE22', # GPIO_1[32]
+                                'AD21', # GPIO_1[33]
+                                'AA20', # GPIO_1[34]
+                                'AC22', # GPIO_1[35]
+                                )),
+
+        # ADC pins (names given in the user manula)
+        #'adc_cs_n': dict(pins=('A10',)),
+        #'adc_saddr': dict(pins=('B10',)),
+        #'adc_sdat': dict(pins=('A9',)),
+        #'adc_sclk': dict(pins=('B14',)),
+
+        # I2C lines shared with accelerometer and EEPROM
+        #'i2c_sclk': dict(pins=('F2',)),
+        #'i2c_sdat': dict(pins=('F1',)),
+        #'g_sensor_cs_n': dict(pins=('G5',)),
+        	#'g_sensor_int': dict(pins=('M2',)),
+
+        # LT24 pins 
+        # @todo: use an extintf interface/object		DE1_SOC Board
+        'lcd_on': dict(pins=('AC22',)),  				# GPIO 1 pin 
+        'lcd_resetn': dict(pins=('AD21',)),				# GPIO 1 pin 
+        'lcd_csn': dict(pins=('AA20',)),				# GPIO 1 pin 
+        'lcd_rs': dict(pins=('AH27',)),					# GPIO 1 pin
+        'lcd_wrn': dict(pins=('AH24',)),				# GPIO 1 pin
+        'lcd_rdn': dict(pins=('AG26',)),				# GPIO 1 pin
+        'lcd_data': dict(pins=('AF26', 'AF25', 'AE24', 'AE23',		# GPIO 1, Data 0:3
+                               'AJ27', 'AK29', 'AK28',  'AK27',		# GPIO 1, Data 4:7
+                               'AJ26', 'AK26', 'AH25', 'AJ25',		# GPIO 1, DAta 8:11
+                               'AJ24', 'AK24', 'AG23', 'AK23',)),	# GPIO 1, DAta 12:15
+
+
+	# VGA Pins added for the Onboard VGA DAC chip. ADV7123
+	# triple 10 Bit High speed Video DAC by analog devices.
+									#Red Data
+	'VGA_R_data': dict(pins=('A13', 'C13', 'E13', 'B12',        # Data 0:3
```

Comment:
don't use upper case here, change to `vga_r_data`, the same for all the following.